src/compojure/api/meta.clj

```diff
@@ -531,14 +532,14 @@
 
 (defmacro static-context
   [path route]
-  `(#'compojure.core/if-context
+  `(#'compojure.api.compojure-compat/make-context
      ~(#'compojure.core/context-route path)
-     ~route))
+     (constantly ~route)))
 
 (defn routing [handlers]
   (if-let [handlers (seq (keep identity handlers))]
-    (apply some-fn handlers)
-    (constantly nil)))
+    (apply compojure.core/routes handlers)
```

Comment:
At some point, I have flattened/inlined some of the compojure function calls. Here, the call stack gets at least 2 extra fn calls. Does not affect perf much thou.